pom.xml

```diff
@@ -51,6 +51,8 @@
         <version>1.0.5</version>
     </parent>
     <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Added to fix issue within IntelliJ that would sometimes revert the compilation to Java 5.